src/thunderbots/software/util/parameter/config/ai_control.yaml

```
@@ -0,0 +1,54 @@
+AI:
+  run_ai:
+    type: "bool"
+    default: false
+    description: >-
+        Selecting will allow HL and Navigator to run, unselecting will
+        stop new decisions from being made, but the robots will finish
+        the last commands they were given. DO NOT USE inplace of the e-stop.
+  override_ai_play:
+    type: "bool"
+    default: false
+    description: >-
+        Overrides the current play with the play
+        specified by current_play parameter
+  current_ai_play:
+    type: "string"
+    default: "spin"
```

```suggestion
    default: "Example Play"
```

Since that's the only valid Play right now.